ops-manager/docker-compose.yml

```diff
@@ -25,6 +25,7 @@ services:
       - "8080:8080"
       - "8443:8443"
       - "25999:25999"
+      - "27017:27017"
```

Comment:
This can be removed, it is irrelevant for the ldap, i used it in a repro and i guess it got left in the code. my bad